pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerBookieIsolationTest.java

```diff
@@ -264,6 +264,152 @@ public void testBookieIsolation() throws Exception {
         assertEquals(clientConf.getProperty(REPP_DNS_RESOLVER_CLASS), ZkBookieRackAffinityMapping.class.getName());
     }
 
+    @Test
```

Comment:
can you please add a comment that explains why this test is different ? in one line what it is "strict bookie isolation"